### ISSUE TITLE
[Unity] Relation Query Fails with error on iOS.

### ISSUE THREAD

**Issue by aquaGhost1** [ParsePlatform/Parse-SDK-dotNET#60]
Error: Query a relation: Must specify a ParseObject class name when creating a ParseQuery.

Unity: We are using 4.6.7 but it also happens on 4.6.8.
Parse plugin 1.6.1 for Unity.
Xcode 6.4

When Querying a relation, we started getting this error after updating parse. 
1. It only happens on iOS, and not in the Unity editor. Query also works in API console.
2. Did not happen in older version of Parse.

We are querying a relation in this manner:
var FBFriendsRelation = ParseUser.CurrentUser.GetRelation<ParseUser> ("FBFriends");
			if (FBFriendsRelation != null) {
				ParseQuery<ParseUser> q = FBFriendsRelation.Query;

In Xcode, the error happens on the final line. We tried this with a stub project that only does this query, and get the same result.

**Comment by kostub** [comment]
I think this is the same as #72.

**Comment by richardjrossiii** [comment]
Yep, I think you're correct. Currently my unity installation isn't fully working so I can't test ATM, but I'm going to close this for now. Feel free to re-open it if the problem persists.

**Comment by aquaGhost1** [comment]
@richardjrossiii still occurring on 1.7.0.

**Comment by richardjrossiii** [comment]
I'll take another look at this.

**Comment by skumancer** [comment]
I'm getting this error as well when switching to IL2CPP. With Mono 2.0, relations work properly.
